serverscommands/instancecommands/rebuild.go

```diff
@@ -67,17 +71,35 @@ func commandRebuild(c *cli.Context) {
 	opts := osServers.RebuildOpts{
 		ImageID:    c.String("imageID"),
 		AdminPass:  c.String("adminPass"),
-		Name:       c.String("name"),
 		AccessIPv4: c.String("accessIPv4"),
 		AccessIPv6: c.String("accessIPv6"),
 	}
 
+	if c.IsSet("rename") {
+		opts.Name = c.String("rename")
+	}
+
 	if c.IsSet("metadata") {
 		opts.Metadata = util.CheckKVFlag(c, "metadata")
 	}
 
 	client := auth.NewClient("compute")
 	serverID := idOrName(c, client)
+
+	if c.IsSet("rename") {
+		opts.Name = c.String("rename")
+	} else if c.IsSet("name") { // If name specified, assume
+		opts.Name = c.String("name")
```

Comment:
Maybe I shouldn't leave this here, so there's not an accidental side effect rename here either.